Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AssetBuilderSDK/AssetBuilderSDK.h>
+
+namespace ROS2
+{
+    struct SdfAssetBuilderSettings
+    {
+    public:
+        AZ_RTTI(ROS2::SdfAssetBuilderSettings, "{7FE3FA92-E274-4624-8905-61E1587DDD30}");
```

Comment:
The `ROS2::` should be removed from the AZ_RTTI macro.
It complicates creating an SdfAssetBuilderSettings object if it was ever reflected to the BehaviorContext as well if we ever need to change the Type Info name from the C++ type
```suggestion
        AZ_RTTI(SdfAssetBuilderSettings, "{7FE3FA92-E274-4624-8905-61E1587DDD30}");
```